android/automated_build/install-qt.sh

```diff
@@ -233,13 +233,13 @@ DOWNLOAD_DIR=`mktemp -d 2>/dev/null || mktemp -d -t 'install-qt'`
 function compute_url(){
     local COMPONENT=$1
     local CURL="curl -s -L"
-    local BASE_URL="http://download.qt.io/online/qtsdkrepository/${HOST_OS}/${TARGET_PLATFORM}"
+    local BASE_URL="https://mirrors.ocf.berkeley.edu/qt/online/qtsdkrepository/${HOST_OS}/${TARGET_PLATFORM}"
```

Comment:
Not sure if we're allowed to change it here (not looked into the license yet). But can you change the mirror here?